src/server/utils/push.util.js

```diff
@@ -0,0 +1,66 @@
+import FCM from 'fcm-push';
+import mongoose from 'mongoose';
+
+const serverKey = 'AAAATC_Jnfw:APA91bE6u80DCTiBgZdI8Fkuc6RHzblvKfWw4gTmFc1aT7Ud1eyGtaSplg8wa6UwOO4' +
```

Comment:
Hmm... I didn't consider!! but as we managed our keys in our key DB, I'll move it! Thanks!